.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+This notebook checklist has been made available to us by the [the Notebooks For All team](https://github.com/Iota-School/notebooks-for-all/blob/main/resources/event-hackathon/notebook-authoring-checklist.md).
+Its purpose is to serve as a guide for both the notebook author and the technical reviewer highlighting critical aspects to consider when striving to develop an accessible and effective notebook.
+
+### The First Cell
+
+- [ ] The title of the notebook in a first-level heading (eg. `<h1>` or `# in markdown`).
+- [ ] A brief description of the notebook.
+- [ ] A title in the form of a H1 (`#` in Markdown).
+- [ ] A brief summary of the notebook.
```

Comment:
What is the difference between a brief description and a brief summary? I feel like if there is a description and a table of content, the summary might be a repetition.